documentation/projects/proposals/additional_search_views/20230424-project_proposal_additional_search_views.md

```diff
@@ -0,0 +1,36 @@
+# 2023-04-24 Project Proposal
+
+## Reviewers
+
+- [ ] @obulat
+- [ ] @krysal
+
+## Project summary
+
+The Openverse team has decided to enhance the browsing experience by
+incorporating new content views for providers, tags, and creators. This feature
+will allow users to share collections detached of individual searches.
+
+## Goals
+
+Search Experience
+
+## Requirements
+
+## Success
+
+The project will be considered successful when each tag, provider, and creator
+has a standalone and shareable page
+
+## Participants and stakeholders
+
+- Design: @panchovm
+- Implementation:
+
+## Infrastructure
+
+## Accessibility
```

Comment:
A call for the a11y team of the WordPress project for revision of the mockups would be nice here.

Reply:
Good call. Based on how the discussion unfolded, I will ping a11y folks in the design ticket.